elements/pfe-sass/functions/_custom-properties.scss

```diff
@@ -234,3 +163,25 @@
       @return var(--#{$repo}-broadcasted--#{to-string($broadcast,'--')}#{unquote($fallback)});
     }
 }
+
+@function pfe-fetch($name) {
```

Comment:
👏  🦜 🎉

Reply:
should we add a comment to the top of each file in `pfe-sass/maps` that says, "This map file has been added to the pfe-fetch function` or something like that?

Reply:
I would say no only because the pfe-var function also does this.  At least not a comment on the map files.  I should add documentation for pfe-fetch though.